src/main/java/net/snowflake/client/jdbc/ArrowResultChunk.java

```diff
@@ -304,9 +304,9 @@ else if (!customMeta.isEmpty())
                 converters.add(new BigIntToTimeConverter(vector, i, context));
                 break;
               default:
-                throw new SnowflakeSQLException(
+                throw new SnowflakeSQLLoggedException(
                     SqlState.INTERNAL_ERROR,
-                    ErrorCode.INTERNAL_ERROR.getMessageCode(),
+                    ErrorCode.INTERNAL_ERROR.getMessageCode(), null,
```

Comment:
same here. We can pass the session to here too.

Reply:
added session